app/infrastructure/queue_consumer.py

```
@@ -162,7 +162,30 @@ async def _dequeue_job(
+            try:
+                await tracker.update_step(JobLevels.PROCESSING)
+            except Exception:
+                # don't fail job if bookkeeping fails
+                self.logger.warning("tracker.update_step(PROCESSING) failed", exc_info=True)
+    
+    async def _set_track_tracer_finished(self, tracker, tracer):
+        if tracker:
+            try:
+                await tracker.update_step(JobLevels.QUEUE_ACK)
+                await tracker.completed()
+            except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)